src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +181,907 @@
+
+(h/deftest-event :wallet/select-address-tab
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:select-address-tab "tab"}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send nil}}})
+    (is (match? expected-db (:db (dispatch [event-id "tab"]))))))
+
+(h/deftest-event :wallet/clean-send-address
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:to-address  "0x01"
+                          :recipient   {:recipient-type :saved-address
+                                        :label          "label"}
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :to-address)))
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :recipient)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-send-amount
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:amount      10
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :amount)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-disabled-from-networks
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                          :other-props             :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :disabled-from-chain-ids)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/clean-from-locked-amounts
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:from-locked-amounts "value"
+                          :other-props         :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :from-locked-amounts)))
+  (is (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :other-props)))
+
+(h/deftest-event :wallet/disable-from-networks
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:disabled-from-chain-ids [:optimism]
+                                          :other-props             :value}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props :value}}}})
+    (is (match? expected-db (:db (dispatch [event-id [:optimism]]))))))
+
+(h/deftest-event :wallet/unlock-from-amount
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:other-props         :value
+                                          :from-locked-amounts {}}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send {:other-props         :value
+                            :from-locked-amounts {:chain-id [1 10]}}}}})
+    (is (match? expected-db (:db (dispatch [event-id :chain-id]))))))
+
+(h/deftest-event :wallet/lock-from-amount
```

Comment:
@VolodLytvynenko - this test is not needed any more right?

Reply:
it seems @vkjr this is addressed to you.

Reply:
Sorry! @vkjr was the intention 😅